packages/test-in-console/puppeteerRunner.js

```diff
@@ -0,0 +1,71 @@
+const puppeteer = require('puppeteer');
+
+async function runNextUrl(browser) {
+  const page = await browser.newPage();
+
+  page.on('console', msg => {
+    console.log(msg._text);
+  });
+
+  if (!process.env.URL) {
+    await page.close();
+    await browser.close();
+    process.exit(0);
+    return;
+  }
+
+  await page.goto(process.env.URL);
+
+  async function poll() {
+    if (await isDone(page)) {
+      let failCount = await getFailCount(page);
+      if (failCount > 0) {
+        await page.close();
+        await browser.close();
+        process.exit(1);
+      } else {
+        await page.close();
+        setTimeout(runNextUrl, 1000);
```

Comment:
I suspect that this `runNextUrl` function being called without a `browser` is the likely cause of the failure identified in https://github.com/meteor/meteor/pull/9814#issuecomment-397271476 and shown in [this test run](https://travis-ci.org/meteor/meteor/builds/392210478).